draft-fossati-tls-attestation.md

```diff
@@ -1061,6 +1075,28 @@ the TLS server will terminate the exchange.
 
 TBD.
 
+# Privacy Considerations {#priv-cons}
+
+In this section, we are assuming that the Attester is a TLS client, representing an individual person.
+We are concerned about the potential leakage of privacy sensitive information about that person, such as the correlation of different connections initiated by them.
+
+In background-check mode, the Verifier not only has access to detailed information about the Attester's TCB through Evidence, but it also knows the exact time and the party with whom the secure channel establishment is attempted (i.e., the RP).
+The privacy implications are similar to online OCSP {{-ocsp}}.
+While the RP may trust the Verifier not to disclose any information it receives, the same cannot be assumed for the Attester, which generally has no prior relationship with the Verifier.
+Some ways to address this include:
+
+* Client-side redaction of privacy-sensitive evidence claims,
+* Using selective disclosure (e.g., SD-JWT {{-sd-jwt}} with EAT {{-rats-eat}}),
+* Co-locating the Verifier role with the RP,
+* Utilizing privacy-preserving attestation schemes (e.g., DAA {{-rats-daa}}), or
+* Utilizing Attesters manufactured with group identities (e.g., {{FIDO-REQS}}).
+
+The latter two also have the property of hiding the peer's identity from the RP.
+
+Note that the equivalent of OCSP "stapling" involves using a passport topology where the Verifier's involvement is unrelated to the TLS session.
+
+Due to the inherent asymmetry of the TLS protocol, if the Attester acts as the TLS server, a malicious TLS client could potentially retrieve sensitive information from attestation evidence and then terminate the connection before disclosing any information to the peer.
```

Comment:
I am not clear about the context of the following text:
`before disclosing any information to the peer`.

In my view the para should terminate at `then terminate the connection`

Reply:
That sounds a bit better, but I'm ok with the current form as well.

Reply:
> I am not clear about the context of the following text: `before disclosing any information to the peer`.
> 
> In my view the para should terminate at `then terminate the connection`

why would you drop the _"[b]efore ... peer"_ ?

Reply:
Did you mean before disclosing its own identity to the Attester?

Reply:
or someone else?

Reply:
> Did you mean before disclosing its own identity to the Attester?
> or someone else?

I don't understand the question.  There are only two peers in a TLS exchange.

Reply:
I think the ambiguity stems from the term `any information`.

If I understand correctly, you intended to state,:
`sensitive information from attestation evidence and then terminate the connection before disclosing its identity to the peer`

Reply:
It's identity in a wider sense, including any hardware/software info that may come from attestation if the peer requested it in SH.

Reply:
Can I make a suggestion? I think the issue isn't when (or even whether) the client drops the connection, it could continue the handshake and just give bogus credentials.. So we can sidestep the client's actions.

```suggestion
Due to the inherent asymmetry of the TLS protocol, if the Attester acts as the TLS server, a malicious TLS client could potentially retrieve sensitive information from attestation evidence without its trustworthiness first being established by the attester.
```

Reply:
True, good catch!